src/apps/containers/components/ContainerReleasedVersions.tsx

```diff
@@ -146,6 +154,27 @@ const ContainerReleasedVersions: React.FC<Props> = ({
       )
     }
   };
+    const copyDictionary = () => {
+        if(version.released && type === "Dictionary"){
+            return (
+                <MenuItem onClick={handleCloseMenu}>
+                    <CopyToClipboard
```

Comment:
I'm really confused why we're putting this on the users clipboard? Does this serve any functional purpose?

Reply:
Not really, that was some testing we forgot to remove, we will remove it.